### PR TITLE
🎩 new logo & favicon

### LINKED ISSUE #1136: Specialized Twitter Cards for each view
@xylodrone could help us on this page.

We can use the current template which is this one [on Series](https://kodadot.xyz/series-insights) 
image

Let's use this as a template and add text under it.
- [Series](https://kodadot.xyz/series-insight) should have "Series Insights"
  - Title: Series Insights
  - Description: Ape into NFTs based on data points
- [Spotlight](https://kodadot.xyz/spotlight) -- "Creators Directory"
  - NFT Artist Rank
  - Discover new artists based on rank
- [Collections](https://kodadot.xyz/rmrk/collections) -- "Collections"
  - Collections 
  - Collect new mints with collection insights
- [Gallery](https://kodadot.xyz/rmrk/gallery) -- "Explore new mints"
  - Explorer
  - Ape into newest mints

### LINKED ISSUE #1207: Add kodadot v3 logo
Add it to `main` and `main-nuxt` 

Replace old logo where needed
The best would be if we can have SVG, but this works for now! :)
Then we can update branding on Twitter &s

[UI kit for kodadot logo v3](https://www.figma.com/file/XEiqGUitNgIFE0AIgA6XAx/KodaDot_Design-Kit?node-id=40%3A107) 

image

v2 just for the records 😄
image

### LINKED ISSUE #1529: Update social media cards
- Collections - https://beta.kodadot.xyz/rmrk/collections
image

- Spotlight - https://beta.kodadot.xyz/spotlight
image

- Gallery feed - https://beta.kodadot.xyz/rmrk/gallery
image

- on all Create dialogs - https://beta.kodadot.xyz/rmrk/create - https://beta.kodadot.xyz/rmrk/mint
image

- Series Insights - https://beta.kodadot.xyz/series-insight
image

- Default elsewhere, like at / &c
image
 
### Ref
#1612

### PR DESCRIPTION
**Pull request by roiLeo**
New branch, new me

### PR type
- [x] Feature

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### What's new?
- [x] closes #1207 
- [x] closes #1136 
- [x] closes #1529 
- [x] changed light text because last color burned my eyes

### Screenshot v3
Screenshot 2021-11-24 at 09-43-17 KodaDot - Kusama NFT Market explorer

### CODE REVIEW

**Comment by roiLeo** [comment]
#### KODADOT V3?
Capture d’écran 2021-11-23 à 4 39 23 PM

**Comment by vikiival** [comment]
> #### KODADOT V3?
> Capture d’écran 2021-11-23 à 4 39 23 PM

❤️ ❤️ ❤️ ❤️ 
Yes please, but for navbar logo please use the small one (K.)
Moreover I would make the logo smaller

**Comment by yangwao** [comment]
For navbar, I would use probably this first one as in that small you would not see the texture
Link for faster extraction from figma https://www.figma.com/file/XEiqGUitNgIFE0AIgA6XAx/KodaDot_Design-Kit?node-id=0%3A1
image

**Comment by yangwao** [comment]
This is also option 

image

**Comment by roiLeo** [comment]
> For navbar, I would use probably this first one as in that small you would not see the texture

Screenshot 2021-11-24 at 09-22-27 KodaDot - Kusama NFT Market explorer

That was painful to export. SVG not working well.
- Which one should I take for favicon? 
- Wich logo for images placeholder? 
- Should I change meta image too?

### Test
#### Current
Frame 4

#### Current without sharp border
Frame 4(1)

**Comment by vikiival** [comment]
You do not want a file with name `assets/K..svg`. 
Actually do you use it somewhere?

**Comment by roiLeo** [comment]
> You do not want a file with name `assets/K..svg`. Actually do you use it somewhere?

Nope, wanted to export logo as svg but didn't worked well, I'm removing it.
Will change logo name for sure.

**Comment by yangwao** [comment]
@roiLeo let's merge recent `main-nuxt` to update preview and push this forward 

re this https://github.com/kodadot/nft-gallery/issues/1355#issuecomment-984000035

**Comment by roiLeo** [comment]
> @roiLeo let's merge recent `main-nuxt` to update preview and push this forward
> 
> re this [#1355 (comment)](https://github.com/kodadot/nft-gallery/issues/1355#issuecomment-984000035)

Updated!

Need feedback on this:
> - Which one should I take for favicon?
> - Wich logo for images placeholder?
> - Should I change meta images too with new one?

**Comment by vikiival** [comment]
Ready to merge? 👁️ 👅 👁️

**Comment by roiLeo** [comment]
@vikiival tested with larger logo, what do you think?
Screenshot 2021-12-24 at 09-41-34 nft-gallery-nuxt

Still go issue with exporing "K." logo to svg

File: components/rmrk/Gallery/Gallery.vue [hunk 1]
```diff
@@ -114,36 +114,6 @@ const components = {
 }
 
 @Component<Gallery>({
```
Comment (vikiival) [inline components/rmrk/Gallery/Gallery.vue:116]: Why is meta gone?

**Review by vikiival — APPROVED** [review]
LFG 🚀

**Comment by vikiival** [comment]
Screenshot 2021-12-27 at 09 39 53
Looks like the build fail

**Comment by roiLeo** [comment]
I wanted to delete unused component but founds out it's been used by statemine & westmint

**Comment by yangwao** [comment]
wait I need to go through colors check, just replace logo not adjusting colors :D

**Comment by yangwao** [comment]
Re colours continue here https://github.com/kodadot/nft-gallery/issues/1638

and pls use svg whenever we can

**Pull request merged**